sdk/keyvault/keyvault-certificates/test/internal/userAgent.spec.ts

```diff
@@ -23,7 +23,7 @@ describe("Certificates client's user agent (only in Node, because of fs)", () =>
     try {
       // The unit-test script has this test file at: test/internal/userAgent.spec.ts
       const fileContents = JSON.parse(
-        fs.readFileSync(path.join(__dirname, "../package.json"), { encoding: "utf-8" })
+        fs.readFileSync(path.join(__dirname, "../../package.json"), { encoding: "utf-8" })
```

Comment:
huh, I wonder why it was passing before.

Reply:
likely the bundled nodejs test goes through the `catch` block below?